### PR TITLE
Feat(import) add check and tests

### PR DESCRIPTION
**Pull request by Pierre-Narcisi**

### CODE REVIEW

File: backend/geonature/core/imports/checks/sql/nomenclature.py [hunk 1]
```diff
@@ -149,3 +150,25 @@ def check_nomenclature_source_status(imprt, entity, source_status_field, ref_bib
             transient_table.c[ref_biblio_field.dest_field] == None,
         ),
     )
+
+
+def check_nomenclature_technique_collect(
+    imprt, entity, source_status_field, technical_precision_field
+):
+    transient_table = imprt.destination.get_transient_table()
+    other = TNomenclatures.query.filter(
```
Comment (jacquesfize) [inline backend/geonature/core/imports/checks/sql/nomenclature.py:159]: A passer en SQLA 2.0 😸

Comment (edelclaux) [reply backend/geonature/core/imports/checks/sql/nomenclature.py:159]: he's back !

File: backend/geonature/tests/imports/files/occhab/valid_file.csv [hunk 2]
```diff
@@ -1,40 +1,42 @@
...
+Import d’un habitat dans une station existante ailleurs dans le fichier, référencé par UUID.;Pas de station;OK !;;462d385f-489a-436b-babb-8cca5fc62e1d;;;;Toto;;;;;prairie;24;;
+Import d’un habitat dans une station existante ailleurs dans le fichier, référencé par id origine.;Pas de station;OK !;Station 1;;;;;Toto;;;;;prairie;24;;
+On importe une station qui existe déjà en base : elle est ignorée;SKIP_EXISTING_UUID;Pas d’habitat;;EXISTING_STATION_UUID;;17/11/2023;17/11/2023;Toto;;POINT(3.634 44.399);St;;;;;
+On importe une station qui existe déjà en base et un nouvel habitat, seul l’habitat est importé;SKIP_EXISTING_UUID;OK !;;EXISTING_STATION_UUID;;17/11/2023;17/11/2023;Toto;;POINT(3.634 44.399);St;;prairie;24;;
+On importe une station et un habitat existant déjà en base;SKIP_EXISTING_UUID;SKIP_EXISTING_UUID;;EXISTING_STATION_UUID;;17/11/2023;17/11/2023;Toto;;POINT(3.634 44.399);St;EXISTING_HABITAT_UUID;prairie;24;;
+Il y a ni UUID, ni id_origine : l’UUID sera généré.;OK !;OK !;;;VALID_DATASET_UUID;17/11/2023;17/11/2023;Toto;;POINT(3.634 44.399);St;;prairie;24;;10
```
Comment (bouttier) [inline backend/geonature/tests/imports/files/occhab/valid_file.csv:41]:
```suggestion
technique collect vaut « autre » mais pas de précision fournise;OK !;CONDITIONAL_MANDATORY_FIELD_ERROR;;;VALID_DATASET_UUID;17/11/2023;17/11/2023;Toto;;POINT(3.634 44.399);St;;prairie;24;;10
```

File: backend/geonature/tests/imports/files/occhab/valid_file.csv [hunk 3]
```diff
@@ -1,40 +1,42 @@
...
+Il y a ni UUID, ni id_origine : l’UUID sera généré.;OK !;OK !;;;VALID_DATASET_UUID;17/11/2023;17/11/2023;Toto;;POINT(3.634 44.399);St;;prairie;24;squalala;10
```
Comment (bouttier) [inline backend/geonature/tests/imports/files/occhab/valid_file.csv:42]:
```suggestion
technique collect vaut « autre » et une précision est bien fournies;OK !;OK !;;;VALID_DATASET_UUID;17/11/2023;17/11/2023;Toto;;POINT(3.634 44.399);St;;prairie;24;moyen précis;10
```

**Review by bouttier — CHANGES_REQUESTED** [review]

**Pull request merged**